static/js/tippyjs.js

```diff
@@ -29,6 +29,7 @@ tippy.setDefaultProps({
     // html content is not supported by default
     // enable it by passing data-tippy-allowHtml="true"
     // in the tag or a parameter.
+    appendTo: "parent",
```

Comment:
This should be moved about the HTML content comment and be given its own comment.